pkgs/by-name/lu/lubelogger/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "lubelogger";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "hargata";
+    repo = "lubelog";
+    rev = "v${version}";
+    hash = "sha256-YxP8XmjYqxuTA2fIV1q7GBsrWPtLr02KkFl4UUNJ2p8=";
+  };
+
+  projectFile = "CarCareTracker.sln";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  makeWrapperArgs = [
+    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelog"
```

Comment:
```
Unhandled exception. System.IO.DirectoryNotFoundException: /nix/store/xrmaqrhii0rglf87jlpavgskydiglm2r-lubelogger-1.2.1/lib/lubelog/
```

```
$ ls /nix/store/xrmaqrhii0rglf87jlpavgskydiglm2r-lubelogger-1.2.1/lib/
lubelogger
```

```suggestion
    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelogger"
```

Reply:
ack, I thought that instance of lubelog was coming from the repo itself, not from the nixpkgs name. will fix